src/plugins/components/PluginsDetailsPage/PluginsDetailsPage.tsx

```diff
@@ -0,0 +1,128 @@
+import Typography from "@material-ui/core/Typography";
+import AppHeader from "@saleor/components/AppHeader";
+import { ConfirmButtonTransitionState } from "@saleor/components/ConfirmButton";
+import Container from "@saleor/components/Container";
+import Form from "@saleor/components/Form";
+import Grid from "@saleor/components/Grid";
+import PageHeader from "@saleor/components/PageHeader";
+import SaveButtonBar from "@saleor/components/SaveButtonBar";
+import { maybe } from "@saleor/misc";
+import { UserError } from "@saleor/types";
+import React from "react";
+
+import i18n from "../../../i18n";
+import PluginInfo from "../PluginInfo";
+import PluginSettings from "../PluginSettings";
+
+export interface FormData {
+  name?: string;
+  description?: string;
+  active: boolean;
+  configuration: Array<{
```

Comment:
I'm not sure if putting so much redundant object data into the form is a good idea.
I'd be cleaner if `configuration` would be of type `ConfigurationItemInput[]`, separating form and view layers.